requirements.txt

```diff
@@ -0,0 +1,4 @@
+pyfrc
```

Comment:
pyfrc isn't required if you're characterizing a Java robot

Reply:
Would you suggest adding additional `requirements.txt` files in one or both of the python directories?

Reply:
Sure. Thinking about it, the ctre example would need `robotpy-ctre` added to the requirements, which surely don't belong at this toplevel.